Makefile

```diff
@@ -10,8 +8,8 @@ build:
 .PHONY: download
 download:
 	@rm -rf hack/libs/${BINARY_NAME}*
-	@cd hack/libs && wget --quiet $(MIRROR)/$(RPM_PATH)/$(IGUAZIO_VERSION)/$(BINARY_NAME).rpm
-	@cd hack/libs && wget --quiet $(MIRROR)/$(DEB_PATH)/$(IGUAZIO_VERSION)/$(BINARY_NAME).deb
+	@cp $(MIRROR)/$(BINARY_NAME).rpm hack/libs
+	@cp $(MIRROR)/$(BINARY_NAME).deb hack/libs
```

Comment:
new phony for `package`. Will not break BC - and will stay true to the name of the operation :)